master/static/srv/get_experiment_labels.sql

```diff
@@ -0,0 +1,4 @@
+SELECT
+    distinct e.config->'labels' AS labels
+FROM
+    experiments e
```

Comment:
nit: we usually keep a new line at the end of the file

Reply:
Still getting used to VSCode having migrated from Vim. One more option to feel at home :)